entry_types/scrolled/config/routes.rb

```diff
@@ -1,5 +1,21 @@
 PageflowScrolled::Engine.routes.draw do
   scope module: 'editor' do
-    resources :sections, only: :create
+    resources :chapters do
+      collection do
+        put :order
+      end
+    end
+
+    resources :sections do
+      collection do
+        put :order
+      end
+    end
+
+    resources :content_elements do
+      collection do
+        put :order
+      end
+    end
```

Comment:
Let's use shallow nested resources to get urls of the form `POST .../sections/5/content_elements` instead of `POST .../content_elements?section_id=5`.

Reply:
Since the `shallow` option is missing, this creates routes of the form `.../sections/:section_id/content_elements/:id` for update and delete actions. The section id is not used, though. Backbone routing will be easier if we only need to work with one level hierarchy. 

`sections` can then also be nested `chapters`.